connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectReattachExecuteHandler.scala

```diff
@@ -30,8 +30,10 @@ class SparkConnectReattachExecuteHandler(
 
   def handle(v: proto.ReattachExecuteRequest): Unit = {
     // An exception will be raised if the session is not available.
-    val _sessionHolder =
+    val sessionHolder =
       SparkConnectService.getIsolatedSession(v.getUserContext.getUserId, v.getSessionId)
+    assert(sessionHolder != null)
```

Comment:
Being pedantic - the compiler might remove _sessionHolder entirely as part of dead code elimination; adding an assertion shall prevent the compiler from doing so.